pkg/vz/vm_darwin.go

```diff
@@ -587,8 +588,11 @@ func attachAudio(driver *driver.BaseDriver, config *vz.VirtualMachineConfigurati
 		config.SetAudioDevicesVirtualMachineConfiguration([]vz.AudioDeviceConfiguration{
 			soundDeviceConfiguration,
 		})
+	case "", "none":
+		return nil
+	default:
+		return fmt.Errorf("unexpected audio device %q", *driver.Yaml.Audio.Device)
```

Comment:
Make sure every `case` ends with a `return` and then move the default out of the `switch` to get rid of the "missing return" error?

```suggestion
		return nil
	case "", "none":
		return nil
	}
	return fmt.Errorf("unexpected audio device %q", *driver.Yaml.Audio.Device)
```

Reply:
You only see these errors when you build for darwin